src/gen/conditions.rs

```diff
@@ -388,13 +395,12 @@ pub struct Spend {
     // coin_amount and puzzle_hash
     pub coin_id: Arc<Bytes32>,
     // conditions
-    // all these integers are initialized to 0, which also means "no
-    // constraint". i.e. a 0 in these conditions are inherently satisified and
-    // ignored. 0 (or negative values) are not passed up to the next layer
-    // One exception is height_relative, where 0 *is* relevant.
-    // the highest height/time conditions (i.e. most strict)
+    // all these integers are initialized to None, which also means "no
+    // constraint". A 0 may impose a constraint (post 1.8.0 soft-fork).
+    // for height_relative, 0 has always imposed a constraint (for ephemeral spends)
```

Comment:
This is probably a good place to drop a comment saying what happens during `(ASSERT_HEIGHT_RELATIVE 0)`